spec.md

```diff
@@ -13,49 +13,23 @@ The goal of this specification is to standardize container image distribution ba
 
 ### Table of Contents
 
-- [Introduction](spec.md)
-- [Notational Conventions](#notational-conventions)
-- [Historical Context](#historical-context)
-- [Scope](#scope)
-  - [Future](#future)
-- [Use Cases](#use-cases)
-  - [Image Verification](#image-verification)
-  - [Resumable Push](#resumable-push)
-  - [Resumable Pull](#resumable-pull)
-  - [Layer Upload De-duplication](#layer-upload-de-duplication)
-- [Changes](#changes)
 - [Overview](#overview)
-  - [Errors](#errors)
-  - [API Version Check](#api-version-check)
-  - [Content Digests](#content-digests)
-  - [Pulling An Image](#pulling-an-image)
-  - [Pushing An Image](#pushing-an-image)
-  - [Listing Repositories](#listing-repositories)
-  - [Listing Image Tags](#listing-image-tags)
-  - [Deleting an Image](#deleting-an-image)
-- [Detail](#detail)
-  - [Errors](#errors-2)
-  - [Base](#base)
-  - [Tags](#tags)
-  - [Manifest](#manifest)
-    - [GET Manifest](#get-manifest)
-    - [PUT Manifest](#put-manifest)
-    - [DELETE Manifest](#delete-manifest)
-  - [Blob](#blob)
-    - [GET Blob](#get-blob)
-      - [Fetch Blob](#fetch-blob)
-      - [Fetch Blob Part](#fetch-blob-part)
-    - [DELETE Blob](#delete-blob)
-  - [Initiate Blob Upload](#initiate-blob-upload)
-    - [POST Initiate Blob Upload](#post-initiate-blob-upload)
-      - [Initiate Monolithic Blob Upload](#initiate-monolithic-blob-upload)
-      - [Initiate Resumable Blob Upload](#initiate-resumable-blob-upload)
-      - [Mount Blob](#mount-blob)
-  - [Blob Upload](#blob-upload)
-    - [GET Blob Upload](#get-blob-upload)
-    - [PATCH Blob Upload](#patch-blob-upload)
-    - [PUT Blob Upload](#put-blob-upload)
-    - [DELETE Blob Upload](#delete-blob-upload)
+	- [Introduction](#introduction)
+	- [Historical Context](#historical-context)
+- [Definitions](#defintions)
+	- [Common Terms](#introduction)
+	- [Document Language](#document-language)
```

Comment:
@rchincha Given your comments  regarding moving "document anguage" (MUST etc.), what are your thoughts on removing the top-level "Definitions" section entirely, instead adding a "Definitions" subsection under overview containing what's currently under "Common Terms" ?

So something more like:
```
- [Overview](#overview)
	- [Introduction](#introduction)
	- [Historical Context](#historical-context)
	- [Definitions](#defintions)	
- [Conformance](#conformance)
	- [Document Language](#document-language)
	- [Minimum Requirements](#minimum-requirements)
```

"Document Language" might not be the correct way to phrase it if its scoped to just the conformance section, but I think you get my point

Reply:
Because there is not much difference between "Definitions" and "Common Terms" to warrant a separate section for each? Makes sense. "Document Language" should what MUST, MAY, etc language means in the document. That should be one of the very first sections before anything else begins, IMO.